shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/di/koin/modules/ViewModelModule.kt

```diff
@@ -4,12 +4,13 @@ import co.nimblehq.blisskmmic.presentation.modules.login.LoginViewModel
 import co.nimblehq.blisskmmic.presentation.modules.resetpassword.ResetPasswordViewModel
 import co.nimblehq.blisskmmic.presentation.modules.splash.SplashViewModel
 import co.nimblehq.blisskmmic.presentation.modules.surveyselection.SurveySelectionViewModel
+import org.koin.core.module.dsl.factoryOf
 import org.koin.core.module.dsl.singleOf
 import org.koin.dsl.module
 
 val viewModelModule = module {
 
-    singleOf(::LoginViewModel)
+    factoryOf(::LoginViewModel)
```

Comment:
@blyscuit Any reason for changing from `Single` to `Factory`?

Reply:
The ViewModel should be re-created once per view, the original [Koin](https://insert-koin.io/docs/quickstart/android#displaying-user-with-presenter) description also does the same.

Reply:
@blyscuit Thanks for your clarification